src/screens/components/Testimonials.js

```diff
@@ -2,49 +2,84 @@ import React from 'react'
 import styled from 'styled-components'
 
 import { TestimonialCard, Title } from '@/style-guide'
+import { secondary1, accent2 } from '@/style-guide/colors'
 
 const StyledContainer = styled.div`
+  background-color: ${secondary1};
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+`
+
+const StyledTestimonials = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
+
   @media (min-width: 760px) {
     flex-direction: row;
     justity-content: space-between;
     align-content: center;
   }
 `
 
+const StyledFooter = styled.div`
+  height: 100px;
+  width: 100%;
+  background-color: ${accent2};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Hammersmith One";
+
+  p {
+    font-weight: bold;
+    color: ${secondary1};
+  }
+`
+
 const Testimonials = () => (
   <>
-    <Title>
-      WHAT OUR CLIENTS ARE SAYING
-    </Title>
     <StyledContainer>
```

Comment:
same thing about background of text here. can't tell if it's how the browser renders the color or if the opacity is turned down (see attached from her mockup:

![Screen Shot 2019-05-24 at 6 02 20 PM](https://user-images.githubusercontent.com/17219379/58358739-28e29e80-7e4e-11e9-8bc6-87678596e827.png))

Reply:
I thinks it's depends os the screen, because shouldn't have any opacity here 🤔  
Take a look at my screen:
![image](https://user-images.githubusercontent.com/18172307/58370858-d226a480-7ee1-11e9-8680-b67b990b0f1f.png)